### PR TITLE
[shell] Fix BE unit test directory not match bug

### PR DESCRIPTION
**Pull request by acelyc111**

### CODE REVIEW

**Comment by acelyc111** [comment]
I introduced this bug by #3990  :(

File: run-ut.sh [hunk 1]
```diff
@@ -105,7 +105,7 @@ echo "    Running PaloBe Unittest    "
 echo "******************************"
 
 cd ${DORIS_HOME}
-export DORIS_TEST_BINARY_DIR=${DORIS_HOME}/be/ut_build
+export DORIS_TEST_BINARY_DIR=${DORIS_HOME}/be/ut_build_${CMAKE_BUILD_TYPE}
```
Comment (morningman) [inline run-ut.sh:108]:
```suggestion
export DORIS_TEST_BINARY_DIR=${CMAKE_BUILD_DIR}
```

**Review by morningman — APPROVED** [review]
LGTM

**Comment by morningman** [comment]
I merged this PR so that following UT can run normally.

**Pull request merged**